### PR TITLE
feat: add getter for data request claimer pkh

### LINKED ISSUE #72: return address of identity that claimed a data request
This issue helps the bridge identify whether it claimed the data request if it goes errors ocurred between claimed-included states. It should provide a getter for the address of the claimer

### PR DESCRIPTION
**Pull request by igorline**
Closes #72

### CODE REVIEW

**Comment by igorline** [comment]
@girazoki, what can I do to make the build pass? Errors are not related to this PR

**Comment by mariocao** [comment]
Hi @igorline, I just pushed a commit with some rule changes in `solhint` that should fix it! :smile:

**Comment by igorline** [comment]
@mariocao thank you! yet waiting for the feedback from @girazoki

File: contracts/WitnetRequestsBoard.sol [hunk 1]
```diff
@@ -215,6 +215,14 @@ contract WitnetRequestsBoard is WitnetRequestsBoardInterface {
     return validIds;
   }
 
+  /// @dev Returns data request claimer pkh.
+  /// @param _id Id of data request identifiers to be checked.
+  /// @return A pkh of the data request claimer.
```
Comment (mariocao) [inline contracts/WitnetRequestsBoard.sol:220]:
```suggestion
  /// @param _id The unique identifier of the data request to be checked.
  /// @return The pkh of the data request claimer.
```

File: contracts/WitnetRequestsBoard.sol [hunk 2]
```diff
@@ -215,6 +215,14 @@ contract WitnetRequestsBoard is WitnetRequestsBoardInterface {
...
+  function getDataRequestPkhClaim(uint256 _id) external view returns (address) {
+      return requests[_id].pkhClaim;
+  }
+
+
```
Comment (mariocao) [inline contracts/WitnetRequestsBoard.sol:225]:
```suggestion
```

File: contracts/WitnetRequestsBoard.sol [hunk 3]
```diff
@@ -215,6 +215,14 @@ contract WitnetRequestsBoard is WitnetRequestsBoardInterface {
     return validIds;
   }
 
+  /// @dev Returns data request claimer pkh.
```
Comment (girazoki) [inline contracts/WitnetRequestsBoard.sol:218]:
```suggestion
  /// @dev Returns the pkh of the data request claimer.
```

**Comment by igorline** [comment]
@girazoki @mariocao thank you! I've updated getter method spec

**Comment by girazoki** [comment]
Sorry for accepting that late, ready to be merged!

**Pull request merged**